java/vector/src/main/java/org/apache/arrow/vector/validate/ValidateVectorVisitor.java

```diff
@@ -90,6 +91,11 @@ public Void visit(BaseVariableWidthVector vector, Void value) {
     return null;
   }
 
+  @Override
+  public Void visit(BaseLargeVariableWidthVector left, Void value) {
+    return null;
```

Comment:
is this another TODO? Would you mind making JIRAs for this and the other TODOs here so they can be tracked?

Reply:
Sure. I will fix this problem in ARROW-8402, and will open other JIRAs to track other TODOs. Thanks for your reminder.